server/models/handlers/account.js

```diff
@@ -0,0 +1,27 @@
+/* The goal: Take in the user input from the Account Creation form in 
+the front end UI in order to create a user account in the backend*/
+
+async function createUserAccount(firstName, lastName, type, studentID, email, password, phone) {
+    
+    const user = {
+        type: 'student', //note: 'type' is called 'role' in the UI, but refers to either a worker or a student.
+        studentId: 20764242,
+        firstname: 'Victor',
+        lastname: 'Hugo',
+        email: 'victorhugo@gmail.com',
+        phone: "6476442200"
```

Comment:
Was there a particular reason why double quotes was used for phone as opposed to the single quotes used for the values of all other attributes?

Reply:
Hi Amy,
That was my mistake: I have changed these to single quotes in line 12 of server/models/handlers/account.js and line 20 of server/test/accountCreation.test.js

Reply:
Hi Nancy, Awesome thank you for implementing that fix.